Leanplum-SDK/Classes/Managers/Networking/LPRequestFactory.h

```diff
@@ -0,0 +1,9 @@
+#import <Foundation/Foundation.h>
+#import "LPRequesting.h"
+
+@interface LPRequestFactory : NSObject
+
++ (id<LPRequesting>)get:(NSString *)apiMethod params:(NSDictionary *)params;
```

Comment:
I'm not sure why we are using the factory pattern here if we're really just defining static `createGetRequst` and `createPostRequest` methods.